plugins/metrics/message.go

```diff
@@ -12,6 +14,27 @@ var (
 	// Total number of processed messages since start of the node.
 	messageTotalCount atomic.Uint64
 
+	// number of messages in the database at startup
+	initialMessageTotalCountDB atomic.Uint64
```

Comment:
I don't really see a reason why these should be `atomic` too. `measureInitialDBStats` is anyway only called once.